setup.py

```diff
@@ -1,50 +1,43 @@
+import os
 from setuptools import setup, find_packages
 
-setup(
-	# Application name:
-	name="minikerberos",
-
-	# Version number (initial):
-	version="0.0.9",
-
-	# Application author details:
-	author="Tamas Jos",
-	author_email="info@skelsec.com",
-
-	# Packages
-	packages=find_packages(),
-
-	# Include additional files into the package
-	include_package_data=True,
+here = os.path.abspath(os.path.dirname(__file__))
 
+with open(os.path.join(here, 'README.md'), encoding='utf-8') as readme_file:
+    long_description = readme_file.read()
 
-	# Details
-	url="https://github.com/skelsec/minikerberos",
-
-	zip_safe = True,
-	#
-	# license="LICENSE.txt",
-	description="Kerberos manipulation library in pure Python",
-
-	# long_description=open("README.txt").read(),
-	python_requires='>=3.6',
-	classifiers=(
-		"Programming Language :: Python :: 3.6",
-		"License :: OSI Approved :: MIT License",
-		"Operating System :: OS Independent",
-	),
-	install_requires=[
-		'asn1crypto',
-	],
-
-	entry_points={
-		'console_scripts': [
-			'ccacheedit = minikerberos.examples.ccache_editor:main',
-			'kirbi2ccache = minikerberos.examples.kirbi2ccache:main',
-			'ccache2kirbi = minikerberos.examples.ccache2kirbi:main',
-			'ccacheroast = minikerberos.examples.ccacheroast:main',
-			'getTGT = minikerberos.examples.getTGT:main',
-			'getTGS = minikerberos.examples.getTGS:main',
-		],
-	}
+setup(
+    name="minikerberos",
+    version="0.0.9",
+    author="Tamas Jos",
+    author_email="info@skelsec.com",
+    packages=find_packages(),
+    include_package_data=True,
+    test_suite="tests",
```

Comment:
tests look like the test suite however I half-assed the test cases.
Will be changed later but so far this shouldn't be marked as a real test-suite.